crm/modules/member/install.inc.php

```diff
@@ -183,4 +183,16 @@ function member_install($old_revision = 0) {
         $res = mysqli_query($db_connect, $sql);
         if (!$res) crm_error(mysqli_error($res));
     }
+    if ($old_revision < 8) {
```

Comment:
Is $old_revision the right number to check here?  Does it need to check the full version string so it doesn't fire on really old versions that just happen to have this same revision number?  Or am I worrying about nothing?  Or is this actually just the "version" of this specific module in which case I am worrying about nothing?

Reply:
@ramgarden yeah, $old_revision is module-specific, so this is version 8 of the member module